web/src/composebox_typeahead.ts

```diff
@@ -79,9 +79,13 @@ export type LanguageSuggestion = {
     type: "syntax";
 };
 
-type TopicSuggestion = {
+export type TopicSuggestion = {
     topic: string;
     type: "topic_list";
+    // This will be used when we want to only render the stream as an option in the topic typeahead
+    // while having #**stream_name> as the token.
+    is_stream_only: boolean;
+    streamData: StreamPillData;
```

Comment:
Sorry for missing this before, but this should be `stream_data` to match casing for `is_stream_only`